minvws/nl-kat-coordination
# Fix empty vulnerability reports

### Changes
This PR adds a template tag to the vulnerability report, so it doesn't show empty reports anymore.

### Issue link
Closes #3637

### Demo
![afbeelding](https://github.com/user-attachments/assets/58193c10-3921-4d5f-8ef6-3068cf76f851)

### QA notes
- Create a normal vulnerability report with multiple OOIs (with and without vulnerabilities)
- Check if the reports show report data if expected and "No vulnerabilities have been found on this system." if no vulnerabilities are expected. 
- Do the same for an aggregate report

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
